src/utils/schema-utils.js

```diff
@@ -25,6 +25,10 @@ export function getTypeInfo(schema) {
     dataType = IS_MISSING_TYPE_INFO_TYPE;
   }
 
+  if (dataType === 'array' && !schema.format) {
+    schema.format = schema.items.format;
```

Comment:
let's not change the `schema.format` but instead populate `info.arrayItem.format` like we do on line 51 so that the code is consistent.

Reply:
![Screen Shot 2022-09-02 at 1 49 02 PM](https://user-images.githubusercontent.com/28315000/188218668-4ddb6d5a-9423-4904-81a1-e44d1ed39703.png)